README.md

```diff
@@ -222,6 +222,28 @@ In order to do that, just call:
 openAds.resetConnectors()
 ```
 
+## Logging
+
+In order to debug what's happening with the Ads Loading, OpenAds uses [LogLevel](https://github.com/pimterry/loglevel) as it's logging framework.
+Also, it's using the [LogLevel Plugin Prefix](https://github.com/kutuluk/loglevel-plugin-prefix) plugin to configure how the log messages will be displayed.
```

Comment:
"Also, it is using [LogLevel Plugin Prefix](https://github.com/kutuluk/loglevel-plugin-prefix) to configure how log messages will be displayed."

Reply:
:+1: